rcon/2023/index.rkt

```diff
@@ -408,6 +408,26 @@ We discuss the design of a deep learning toolkit, @a[#:href "https://github.com/
 Hardcoding passwords in your programs is bad.  Using secure password stores are
 good.  @a[#:href "https://pkgs.racket-lang.org/package/keyring" #:title "keyring (Racket package)"]{Keyring} is a Racket library that allows programs to access  different
 password stores using a simple interface.
+}
+]
+
+  @lecture[
+#:when
+@talk-time{Saturday, 3:00pm}
+#:who
+@speaker[#:url "https://github.com/samdphillips"]{Robby Findler}
```

Comment:
I think this is probably the wrong url :)

Reply:
Ack, quite right. Fixed!